src/scaffold/services/columns/columns.test.tsx

```diff
@@ -0,0 +1,147 @@
+import { GridEnrichedColDef } from "@mui/x-data-grid";
```

Comment:
these tests + columns file are also likely to change when splitting out design layers